templates/payment/email/confirmation_mail.txt

```diff
@@ -0,0 +1,11 @@
+Hei!
+
+Du har betalt for {{ payment_relation }} på datoen {{ payment_date }}.
```

Comment:
"på datoen" {{ date }}? Høres litt snodig ut.

Se forøvrig på https://www.altinn.no/no/Starte-og-drive-bedrift/Drive/Regnskap-og-revisjon/Hva-er-bokforingsplikt/Hva-skal-sta-pa-salgsdokumentene/ for en liste over andre ting som og bør stå på kvitteringen.

Det hadde heller ikke skadet å formatere den som en vanlig kvittering, sett opp med "vare" og antall hvis det er tilgjengelig og klask på totalsum nederst i en bunnlinje.